frontend/src/components/RecipeList.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+
+function RecipeList() {
+  const [recipes, setRecipes] = useState();
+
+  useEffect(() => {
+    async function getRecipeList() {
+      try {
+        const response = await axios.get('http://localhost:8081/api/recipes');
```

Comment:
Használjazok proxyt és akkor az axiosnál nem kell a http://localhost:8081 prefix. Annál is inkább, mert majd a stage környezetben egy mezei user localhostján nem lesz backend futtatva.
Részletek: https://create-react-app.dev/docs/proxying-api-requests-in-development/
Ha van kérdés, akkor keressetek.

Reply:
const recipeAPI = axios.create({
    baseURL: 'http://localhost:8081/api/',
  });

javítva

Reply:
RTFM, olvasd el a doksit amit küldtem, nem az axios instance a lényeg itt.

Reply:
done ,thx